### ISSUE TITLE
tmuxp assertion error with tmux master (pane index)

### ISSUE THREAD

**Issue by kcwu** [tony/tmuxp#172]
after https://github.com/tmux/tmux/commit/00cf5fbde6d846a10804447204dd55e8c3a68dbd
the pane index may change after window split.

If there are two panes and their ids are 0 and 1. If calling libtmux's split_window() without specifying "target", libtmux will split pane 0. The result indexes are 0,1,2. Which means, new pane's id is 1 and previous pane's id changed from 1 to 2.

In old version, the indexes are 0,2,1.

With this change, all tmuxp sessions, with more than 2 panes in the same window, are probably all broken.

```
$ cat test.yaml                                                                                                                                                                                                                                                                     session_name: test
windows:
        - window_name: test
          panes:
                  - echo 1
                  - echo 2
                  - echo 3
                  - echo 4
```

```
$ tmuxp load test.yaml
[Loading] /usr/local/google/home/kcwu/tmuxp/test.yaml
Traceback (most recent call last):
  File "/home/kcwu/.local/bin/tmuxp", line 9, in <module>
    load_entry_point('tmuxp', 'console_scripts', 'tmuxp')()
  File "/home/kcwu/.local/lib/python2.7/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/home/kcwu/.local/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/home/kcwu/.local/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/kcwu/.local/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/kcwu/.local/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/home/kcwu/.local/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/kcwu/src/tmuxp/tmuxp/cli.py", line 484, in command_load
    load_workspace(config[-1], **tmux_options)
  File "/home/kcwu/src/tmuxp/tmuxp/cli.py", line 260, in load_workspace
    builder.build()
  File "/home/kcwu/src/tmuxp/tmuxp/workspacebuilder.py", line 178, in build
    for p, pconf in self.iter_create_panes(w, wconf):
  File "/home/kcwu/src/tmuxp/tmuxp/workspacebuilder.py", line 290, in iter_create_panes
    assert(int(p.index) == int(pindex))
AssertionError
```

**Comment by tony** [comment]
nice detective work @kcwu !

What version of tmux do you encounter the error on?

**Comment by tony** [comment]
If its 00cf5fb, I guess its just HEAD?

**Comment by kcwu** [comment]
yes, i'm using HEAD.

**Comment by tony** [comment]
Roger.  not in 2.2 and under I assume then?

Interested in taking a try at a fix?

**Comment by kcwu** [comment]
Yes, 2.2 is good. Glad to try a fix.

**Comment by tony** [comment]
Give it a shot @kcwu !

**Comment by tony** [comment]
@kcwu Is that as in you want to make an attempt at writing a fix yourself?

**Comment by kcwu** [comment]
Ah, I misunderstood. I'm glad to try if you have a fix ;)

**Comment by adambiggs** [comment]
Tmux 2.3 is now out, which is what brought me here. There's going to be a lot of others coming here soon 😅

**Comment by tony** [comment]
Give 1.2.2 a shot

**Comment by adambiggs** [comment]
Problem solved in 1.2.2. Thanks @tony!
